simp-protocol-kt/src/main/kotlin/org/example/myhome/simp/core/SimpCodec.kt

```diff
@@ -10,6 +11,7 @@ import java.nio.charset.Charset
 /**
  * Created by alexei on 15.06.17.
  */
+@Sharable
```

Comment:
is this annotation is really needed?
If I understand it correctly this annotation is needed if your handler have state. your looks don't have

Reply:
https://netty.io/4.0/api/io/netty/channel/ChannelHandler.Sharable.html
In our case, handler could be reused in multiple pipelines, because it's stateless.

Reply:
https://netty.io/4.0/api/io/netty/channel/ChannelHandler.Sharable.html
In our case, handler could be reused in multiple pipelines, because it's stateless.